simulation_parameters/Constants.cs

```diff
@@ -46,6 +46,29 @@ public static class Constants
     // Should be the same as its counterpart in shaders/CompoundCloudPlane.shader
     public const float CLOUD_MAX_INTENSITY_SHOWN = 1000;
 
+    // Length of wanderer microbe spawn cycle in seconds.
+    public const float WANDERER_SPAWN_RATE = 60f;
+
+    public const int SPAWN_DENSITY_MULTIPLIER = 400000;
+
+    public const int SPAWN_GRID_SIZE = 100;
+    public const int SPAWN_GRID_HALFSIZE = SPAWN_GRID_SIZE / 2;
+
+    public const int SPAWN_EVENT_MIN = 10;
+    public const int SPAWN_EVENT_MAX = 15;
+
+    public const int SPAWN_GRID_WIDTH = 2;
+
+    public const int DESPAWN_ITEM_RADIUS = 170;
+    public const int DESPAWN_ITEM_RADIUS_SQR = DESPAWN_ITEM_RADIUS * DESPAWN_ITEM_RADIUS;
+    public const int SPAWN_WANDERER_RADIUS = (int)(0.95 * DESPAWN_ITEM_RADIUS);
```

Comment:
Would floating point precision be enough here instead of a double?

Reply:
I moved all of the spawn constants to one place and renamed this one. The old system had spawn radius stored as an int and do not know why. Yeah, I should have changed it.